docs/development/README.md

```diff
@@ -0,0 +1,102 @@
+# Development: Apps on Kind
+
+(I'll integrate this better when I have tested everything.)
+
+~~The `local-path-provisioner` seems to fail after a while~~
+
+~~Currently it seems like `gatekeeper` might break `local-path-provisioner`~~
+
+Containerd failed with something about mounting `/dev/input/event*` device, but it seem to work fine after recreating the kind cluster
+
+Labelling the namespaces seem to have done the trick
+
+## Setup
+
+> [!warning]
+> Ensure your `KUBECONFIG` variable is not pointing to a config that you don't want to get edited by `kind`.
+
+> [!important]
+> Ensure your `inotify` limits are high enough else [pods might fail with the "too many open files" or similar](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files)
+
+```
+kind create cluster --name compliantkubernetes --config docs/development/cluster.yaml
+kind get kubeconfig --name compliantkubernetes > "${CK8S_CONFIG_PATH}/.state/kube_config_sc.yaml"
+kind get kubeconfig --name compliantkubernetes > "${CK8S_CONFIG_PATH}/.state/kube_config_wc.yaml"
+
+kubectl create namespace tigera-operator
+helm -n tigera-operator install tigera ./docs/development/charts/projectcalico/tigera-operator-v3.26.4.tgz
+
+kubectl label namespace calico-apiserver calico-system local-path-storage tigera-operator owner=operator
```

Comment:
```suggestion
kubectl label namespace calico-apiserver calico-system local-path-storage tigera-operator owner=operator --overwrite=true
```

Reply:
Does that still happen with kubectl v1.26? I know it doesn't on v1.28 and I think v1.27.